Tests/BowLaws/DivideLaws.swift

```diff
@@ -0,0 +1,22 @@
+import Bow
+import BowGenerators
+import SwiftCheck
+
+public final class DivideLaws<F: Divide & ArbitraryK & EquatableK> {
+    public static func check(
+        isEqual: @escaping (Kind<F, ((Int, Int), Int)>, Kind<F, (Int, (Int, Int))>) -> Bool
+    ) {
+        ContravariantLaws<F>.check()
```

Comment:
As I mentioned in the `Monoidal` PR, we are not checking transitively the laws in the hierarchy.